prowler/providers/aws/services/ec2/ec2_instance_multiple_eni/ec2_instance_multiple_eni.py

```diff
@@ -0,0 +1,49 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.ec2.ec2_client import ec2_client
+
+
+class ec2_instance_multiple_eni(Check):
+    def execute(self):
+        findings = []
+        for instance in ec2_client.instances:
+            report = Check_Report_AWS(self.metadata())
+            report.region = instance.region
+            report.resource_id = instance.id
+            report.resource_arn = instance.arn
+            report.resource_tags = instance.tags
+            eni_types = {"efa": [], "interface": [], "trunk": []}
+            if not instance.network_interfaces:
+                report.status = "PASS"
+                report.status_extended = (
+                    f"EC2 Instance {instance.id} has no network interfaces attached."
+                )
+            else:
+                for eni_id in instance.network_interfaces:
+                    if ec2_client.network_interfaces[eni_id].type in eni_types:
+                        eni_types[ec2_client.network_interfaces[eni_id].type].append(
+                            eni_id
+                        )
+
+                message_status_extended = ""
+                if len(eni_types["efa"]) + len(eni_types["interface"]) > 1:
```

Comment:
Why not trunk here?

Reply:
SecurityHub recommendation is to check only for multiple EFAs or interfaces. Thats because a trunk ENI is used to increase the number of ENIs you can attach to a instance. For more information look at the first paragraph here https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-instance-eni.html and at the AWSVPC trunking section here https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-account-settings.html. I don't know if there should be a limit to trunk ENIs maybe we have to discuss that.

Reply:
Can we then ignore trunk ENIs in the check? So the user do not get confused when seeing the status extended since he is not going to be able to delete those trunk ENIs.

Reply:
We saw that trunks are normal interfaces with a specific option so I would count it as an ENI too.